lib/ex_webrtc/sdp_utils.ex

```diff
@@ -123,6 +123,13 @@ defmodule ExWebRTC.SDPUtils do
     |> Enum.map(fn {"candidate", attr} -> attr end)
   end
 
+  @spec add_ice_candidates(ExSDP.t(), [binary()]) :: ExSDP.t()
```

Comment:
To be consistent with get_ice_candidates?

```suggestion
  @spec add_ice_candidates(ExSDP.t(), [String.t()]) :: ExSDP.t()
```